core/templates/core/widget_tag_editor.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+
+<div data-tags-url="{% url 'api:api-root' %}tags/"
+    {% for k, v in widget.attrs.items %}
+        {{ k }}="{{ v }}"
+    {% endfor %}>
+    {% csrf_token %}
+    <span class="prototype-tag btn badge badge-pill cursor-pointer mr-1" style="display: none;">
+        UNINITIALIZED PROTOTYPE
+        <span class="add-remove-icon pl-1 pr-1">+ or -</span>
+    </span>
+    <div class="current_tags" style="min-height: 2em;">
+        {% for t in widget.value %}
+        <span data-value="{{ t.name }}" data-color="{{ t.color }}" class="tag btn badge badge-pill cursor-pointer mr-1" style="background-color: {{ t.color }};">
```

Comment:
Lets add a simple `title` attribute to these tags, "Remove tag".

Reply:
I think the title-attribute is the last open issue... I was thinking a bit about it and still wonder a bit what the idea behind it would be: I assume it is to disambiguate between the internal identifier and the shown title, right? If so, shouldn't we choose slug and name instead? Slug is not really used in the frontend however, because I discovered that taggit refers to tags by "name" by default, so they are pretty much a primary-key as well. That is why there is only a "name" at the moment.

I can am not opposed to adding still add `data-title` as well but it would just be equal to `data-name`. Is that acceptable?

Reply:
Really all I’m thinking is there should be some kind of indicator about what clicking on the tags does. It’s OK as-is but the title might just be a helpful addition. I’m not totally attached to it.

Reply:
Ah! So you mean an actually visible ui-element, right?

Reply:
Yeah. `title` is just a cheap/easy way to do that.

Reply:
I substituted the title-idea with the help_text, mainly because I could not get the "title" idea to "look right".

![grafik](https://user-images.githubusercontent.com/542105/157226377-33e24865-2470-47b8-a98c-429df532178f.png)

Could probably be styled better though, but I think it does the trick.